### PR TITLE
CLOUDP-60955: mongocli om logs list

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ [CLOUDP-60955](https://jira.mongodb.org/browse/CLOUDP-60955)

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
 ./bin/mongocli om logs list --verbose true --profile OPS
{
	"links": [
		{
			"rel": "self",
			"href": "http://localhost:8080/api/public/v1.0/groups/5ea2b3a72f3b443e64fb45fe/logCollectionJobs?verbose=true\u0026pageNum=1\u0026itemsPerPage=100"
		}
	],
	"results": [
		{
			"id": "5ea2c9dd2f3b443e64fb727d",
			"groupId": "5ea2b3a72f3b443e64fb45fe",
			"userId": "5ea2b3dd2f3b443e64fb4648",
			"creationDate": "2020-04-24T11:13:33Z",
			"expirationDate": "2020-05-24T11:13:33Z",
			"status": "SUCCESS",
			"resourceType": "replicaset",
			"resourceName": "myReplicaSet",
			"rootResourceName": "myReplicaSet",
			"rootResourceType": "replicaset",
			"downloadUrl": "http://localhost:8080/api/public/v1.0/groups/5ea2b3a72f3b443e64fb45fe/logCollectionJobs/5ea2c9dd2f3b443e64fb727d/download",
			"redacted": false,
			"logTypes": [
				"AUTOMATION_AGENT"
			],
			"sizeRequestedPerFileBytes": 64,
			"uncompressedSizeTotalBytes": 23242,
			"childJobs": [
				{
					"automationAgentId": "5ea2c9dd2f3b443e64fb727f",
					"finishDate": "2020-04-24T11:13:55Z",
					"hostName": "andreaMac",
					"logCollectionType": "AUTOMATION_AGENT",
					"path": "andreaMac/automation_agent",
					"startDate": "2020-04-24T11:13:33Z",
					"status": "SUCCESS",
					"uncompressedDiskSpaceBytes": 23242
				}
			]
		},
		{
			"id": "5ea2b69d2f3b443e64fb4b39",
			"groupId": "5ea2b3a72f3b443e64fb45fe",
			"userId": "5ea2b3dd2f3b443e64fb4648",
			"creationDate": "2020-04-24T09:51:25Z",
			"expirationDate": "2020-05-24T09:51:25Z",
			"status": "SUCCESS",
			"resourceType": "replicaset",
			"resourceName": "myReplicaSet",
			"rootResourceName": "myReplicaSet",
			"rootResourceType": "replicaset",
			"downloadUrl": "http://localhost:8080/api/public/v1.0/groups/5ea2b3a72f3b443e64fb45fe/logCollectionJobs/5ea2b69d2f3b443e64fb4b39/download",
			"redacted": false,
			"logTypes": [
				"AUTOMATION_AGENT"
			],
			"sizeRequestedPerFileBytes": 64,
			"uncompressedSizeTotalBytes": 23242,
			"childJobs": [
				{
					"automationAgentId": "5ea2b69d2f3b443e64fb4b3b",
					"finishDate": "2020-04-24T09:51:54Z",
					"hostName": "andreaMac",
					"logCollectionType": "AUTOMATION_AGENT",
					"path": "andreaMac/automation_agent",
					"startDate": "2020-04-24T09:51:25Z",
					"status": "SUCCESS",
					"uncompressedDiskSpaceBytes": 23242
				}
			]
		}
	],
	"totalCount": 2
}
```

### CODE REVIEW

File: internal/cli/ops_manager_logs_list.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+		return err
+	}
+	return json.PrettyPrint(result)
+}
+
+func (opts *opsManagerLogsListOpts) newLogListOptions() *om.LogListOptions {
+	return &om.LogListOptions{Verbose: opts.Verbose}
+}
+
+// mongocli om logs collect --verbose verbose [--projectId projectId]
```
Comment (andreaangiolillo) [inline internal/cli/ops_manager_logs_list.go:51]:
```suggestion
// mongocli om logs list --verbose verbose [--projectId projectId]
```

File: internal/cli/ops_manager_logs_list.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+	}
+	return json.PrettyPrint(result)
+}
+
+func (opts *opsManagerLogsListOpts) newLogListOptions() *om.LogListOptions {
+	return &om.LogListOptions{Verbose: opts.Verbose}
+}
+
+// mongocli om logs list --verbose verbose [--projectId projectId]
+func OpsManagerLogsListOptsBuilder() *cobra.Command {
```
Comment (gssbzn) [inline internal/cli/ops_manager_logs_list.go:52]: There was a comment in the jira ticket about making this, maybe, `mongocli om log jobs list` we could do that on a separate PR but just letting you know this may change

Comment (andreaangiolillo) [reply internal/cli/ops_manager_logs_list.go:52]: yes I haven’t done it yet because it seems to be something that we still have to decide

Comment (andreaangiolillo) [reply internal/cli/ops_manager_logs_list.go:52]: I've created this ticket https://jira.mongodb.org/browse/CLOUDP-61392 to keep track of this change
